web/client/components/mapviews/settings/CompactRichTextEditor.jsx

```diff
@@ -123,3 +125,8 @@ function CompactRichTextEditor({
 }
 
 export default CompactRichTextEditor;
+
+CompactRichTextEditor.propTypes = {
+    toolbarOptions: PropTypes.array,
+    wrapperClassName: PropTypes.string
+};
```

Comment:
I did it because there is an eslint rule that "if enabled locally" asks to do that. see also [here](https://github.com/geosolutions-it/MapStore2/pull/9188)

in general I do not like spreading props around especially there, you lose visibility and many times you add unneeded attributes to the final dom. so in this case for now I can remove these definitions but there are many components that are missing these propTypes